src/views/TechnicalAssistance/AdminHome/Consult.tsx

```diff
@@ -162,6 +200,7 @@ function Consult() {
                 onChange={val => {
                   field.onChange(val);
                 }}
+                defaultValue={field.value}
```

Comment:
This is indeed weird behavior, but I found explicitly setting the conditional default directly in the component renders the date on refresh.

`defaultValue={field.value || date}`

Reply:
Did you want to hold off on addressing this for the future?  If so we need to create a bug ticket. @adamodd

Reply:
I guess merge this sooner so that people have slightly better ui since it is working, and create the ticket .

It could take a few to test the [datepicker with a default value](https://trussworks.github.io/react-uswds/?path=/story/components-date-picker--with-default-value) in a regular hook form. If that still doesn't work then `useActionForm` would need a closer look.